### ISSUE TITLE
configure formatter max line width

### ISSUE THREAD

**Issue by bolds07** [eclipse/wildwebdeveloper#759]
It is a very simple and trivial task.

I want the auto formatter to allow very long lines,  I hate when formatter breaks function signature because there are too many parameters or any single line instruction gets breaked into multiple lines because the statement is too long

on eclipse java i go to preferences -> java -> editor -> formatter and set the line width to big number like 300

but i've tried to set the line width in every single editor option in eclipse and none of them seems to affect wild web developer
html, javascript, anything....

always when i press control + shift + f the lines are breaking

ex:

```
<!-- Google Fonts -->
	<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i|Playfair+Display:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700|Poppins:300,300i,400,400i,500,500i,600,600i,700,700i" rel="stylesheet">
```
 pressing control + shift + f 

goes to

```
<!-- Google Fonts -->
	<link
		href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i|Playfair+Display:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700|Poppins:300,300i,400,400i,500,500i,600,600i,700,700i"
		rel="stylesheet">
```

where can i adjust this setting

**Comment by angelozerr** [comment]
Once WWD will be released you will able to configure that like this:

![image](https://user-images.githubusercontent.com/1932211/198134143-df96d92a-0f6c-4880-a4b0-f6e0ecdf54da.png)

**Comment by mickaelistria** [comment]
Fixed with #925; this is already available in snspahots that you can try as described in https://github.com/eclipse/wildwebdeveloper/blob/master/CONTRIBUTING.md#-trying-latest-builds
